Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewMenu.kt

```diff
@@ -0,0 +1,39 @@
+package de.rki.coronawarnapp.contactdiary.ui.overview
+
+import android.content.Context
+import android.view.View
+import android.widget.PopupMenu
+import androidx.navigation.NavController
+import androidx.navigation.fragment.findNavController
+import de.rki.coronawarnapp.R
+import javax.inject.Inject
+
+class ContactDiaryOverviewMenu @Inject constructor(
+    private val contactDiaryOverviewFragment: ContactDiaryOverviewFragment
+) {
+    private val context: Context = contactDiaryOverviewFragment.requireContext()
+
+    private val navController: NavController
+        get() = contactDiaryOverviewFragment.findNavController()
+
+    fun showMenuFor(view: View) = PopupMenu(context, view).apply {
+        inflate(R.menu.menu_contact_diary_overview)
+        setOnMenuItemClickListener {
+            return@setOnMenuItemClickListener when (it.itemId) {
+                R.id.menu_contact_diary_information -> {
+                    true
```

Comment:
minor: no need for braces and extra  line  ... each case can be one line for easier readability